<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsView</class>
 <widget class="QWidget" name="ResultsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resultados del Quiz</string>
  </property>
  <property name="styleSheet">
   <string>QWidget {
	background-color: #2e2e2e;
	color: white;
}
QPushButton {
	padding: 15px;
	font-size: 18px;
	border-radius: 8px;
	font-weight: bold;
}
QPushButton#btn_replay {
	background-color: #007bff;
	color: white;
}
QPushButton#btn_ranking {
	background-color: #ffc107;
	color: #2e2e2e;
}
QPushButton#btn_menu {
	background-color: #555555;
	color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>¡QUIZ TERMINADO!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_score">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: #28a745;</string>
     </property>
     <property name="text">
      <string>Puntuación Final: 0 de 10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbl_name_prompt">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingresa tu nombre para guardar tu récord:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_player_name">
     <property name="objectName">
      <string>txt_player_name</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: #444444; color: white; padding: 10px; border-radius: 5px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Tu Nombre (Presiona ENTER para guardar)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_middle">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_replay">
       <property name="objectName">
        <string>btn_replay</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Jugar de Nuevo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ranking">
       <property name="objectName">
        <string>btn_ranking</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Ver Ranking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_menu">
       <property name="objectName">
        <string>btn_menu</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Volver al Menú</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>PySide6.QtWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>